community/kernel/src/main/java/org/neo4j/graphdb/notification/NotificationSeverityLevel.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphdb.notification;
+
+/**
+ * NotificationSeverityLevel indicates to a client the severity of a notification.
+ */
+public enum NotificationSeverityLevel
```

Comment:
Since this class is already in a package called `notification` it seems a bit superfluous to call this type `NotificationSeverityLevel`, just `SeverityLevel` might suffice.